src/osd/ECUtil.h

```diff
@@ -32,6 +32,8 @@ const uint64_t CHUNK_INFO = 8;
 const uint64_t CHUNK_PADDING = 8;
 const uint64_t CHUNK_OVERHEAD = 16; // INFO + PADDING
 
+const std::string HINFO_KEY = "hinfo_key";
```

Comment:
> self-contained header file would be helpful to speed up the compiling, and in general, also helpful to minimize cross dependencies, imho.

OIC. 

I also wanted to make that parameter private. But there is no class but just namespace...

https://github.com/ceph/ceph/blob/master/src/msg/msg_types.h#L40-L46

How about that?

Any suggestion always welcome!